my_game.py

```diff
@@ -96,7 +77,6 @@ def __init__(self, center_x, center_y, lives, scale):
 
         self.speed = 1
         self.angle = 0
-        self.lives = lives
         self.scale = scale
```

Comment:
Let's use _PLAYER_START_LIVES_ for setting_self.lives_. How is _lives_ set? You removed it from _init()_.